Makefile

```diff
@@ -26,6 +26,10 @@ GOBIN ?= $$(go env GOPATH)/bin
 
 build: $(TOPDIR)/bin/amd64-linux/habitat $(TOPDIR)/bin/amd64-darwin/habitat
```

Comment:
We can add the rule for building the CLI here as another pre-requisite, so everything get's rebuilt at once

Reply:
Did this!